### PR TITLE
ddl, metrics: Move DDL metrics from ddl package to metrics package

### PR DESCRIPTION
**Pull request by shenli**
Move code and fix an issue reported by `goword`.
@zimulala @winkyao @coocood PTAL

### CODE REVIEW

**Comment by coocood** [comment]
@shenli 
We don't need to comment on every variable, just comment one line before `var (` will do.

**Comment by shenli** [comment]
@coocood I think it worth the effort to write comments for the exported variable.

File: metrics/ddl.go [hunk 1]
```diff
@@ -11,23 +11,22 @@
...
 // limitations under the License.
 
-package ddl
+package metrics
 
 import "github.com/prometheus/client_golang/prometheus"
 
 var (
-	jobsGauge = prometheus.NewGaugeVec(
+	// JobsGauge is the metrics for waiting ddl job.
```
Comment (zimulala) [inline metrics/ddl.go]: s/ddl/DDL ?
Please unify it.

File: metrics/ddl.go [hunk 2]
```diff
@@ -11,23 +11,22 @@
...
 			Subsystem: "ddl",
 			Name:      "waiting_jobs",
 			Help:      "Gauge of jobs.",
 		}, []string{"action"})
 
-	// operation result state
-	opSucc             = "op_succ"
-	opFailed           = "op_failed"
-	handleJobHistogram = prometheus.NewHistogramVec(
+	// HandleJobHistogram is the metrics for handle ddl job duration.
```
Comment (zimulala) [inline metrics/ddl.go]: Ditto.

**Comment by coocood** [comment]
@shenli 
For metrics variable, there is a help field, we don't need to duplicate it in the comment.
For string variables, it just represent the string literal, we can use it for different places.

**Comment by shenli** [comment]
@coocood OK

**Comment by shenli** [comment]
@coocood @zimulala PTAL

**Comment by coocood** [comment]
LGTM

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**